### PR TITLE
SorrTask-513-clean-up-gsheet_into_pandas-doc

### PR DESCRIPTION
**Pull request by Dylan-Celli**

### CODE REVIEW

**Comment by DanilYachmenev** [comment]
so is it ready for review?
if so, pls treat the PR accordingly

**Review by samarth9008 — APPROVED** [review]
LGTM according to the specs.

File: README.md [hunk 1]
```diff
@@ -140,8 +140,11 @@ This contains a set of resources that over time will make 10x more productive
 
 - [Telegram](/docs/Telegram.md)
 
+- [Gsheet into pandas](/docs/Gsheet_into_pandas.md)
+
 - [Email](/docs/Email.md)
 
+
```
Comment (DanilYachmenev) [inline README.md:147]: pls remove this extra line

**Review by DanilYachmenev — COMMENTED** [review]
all nice, just 1 micro-nit and I'll merge

**Review by gpsaggese — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (Dylan-Celli) [reply README.md]: Done!

**Pull request merged**